W-16153294 Adding Execute excel macro AS (#409)
mulesoft/docs-rpa: rpa-builder/6.7/modules/ROOT/pages/toolbox-excel-operations.adoc

Use Excel Controlled Loop to iterate over data from an Excel file.
* xref:toolbox-excel-operations-excel-session.adoc[Excel Session]
+

> The Excel Session is an element that encompasses Excel operations, except for Execute Excel Macro. Excel operations include the reading, modification, and creation of entire files and file elements in the Excel format.

Reviewer: ```suggestion
The Excel Session is an element that encompasses Excel operations, except for Execute Excel Macro. Excel operations include reading, modifying, and creating entire files and file elements in Excel format.
```

* xref:toolbox-excel-operations-execute-excel-macro.adoc[]
+

> Run a macro in an Excel file or sheet. This action step does not run inside of an Excel Session.

Reviewer: ```suggestion
Run a macro in an Excel file or sheet. This action step doesn’t run inside an Excel Session.
```
